CHANGES.rst

```diff
@@ -11,6 +11,8 @@ dev (master)
 * Add a server_hostname parameter to HTTPSConnection which allows for
```

Comment:
Could you move the "... <Example>" changelist entry to the bottom of the list? I don't know how it crept up there! :)